app/src/pages/prompt/ClonePromptDialog.tsx

```diff
@@ -173,6 +191,39 @@ export const ClonePromptDialog = ({
                     </TextField>
                   )}
                 />
+                <Controller
+                  control={control}
+                  name="metadata"
+                  rules={{
+                    validate: (value) => {
+                      // Allow empty values (will be treated as undefined)
+                      if (!value || value.trim() === "") {
+                        return true;
+                      }
+                      if (!isJSONObjectString(value)) {
+                        return "metadata must be a valid JSON object";
+                      }
+                      return true;
+                    },
+                  }}
+                  render={({
+                    field: { onChange, onBlur, value },
+                    fieldState: { invalid, error },
+                  }) => (
+                    <CodeEditorFieldWrapper
+                      validationState={invalid ? "invalid" : "valid"}
+                      label="Metadata"
+                      errorMessage={error?.message}
+                      description="A JSON object containing metadata for the prompt (leave empty to remove)"
```

Comment:
```suggestion
                      description="A JSON object containing metadata for the prompt"
```

Last part feels unnecessary.